src/wanderer/eightbit.ts

```diff
@@ -0,0 +1,43 @@
+import { eightBitPoints, Location } from "kolmafia";
+import { DraggableFight } from ".";
+import { WandererTarget } from "./lib";
+import { $item, $location, get, have } from "libram";
+import { garboValue } from "../value";
+import { globalOptions } from "../config";
+
+const FAT_LOOT_TOKEN_COST = 20000;
+
+export const bonusColor = ["black", "blue", "green", "red"] as const;
+export type BonusColor = (typeof bonusColor)[number];
+
+// taken from tour guide, who took it from beldur
+
+const locationColor: Record<BonusColor, Location> = {
+  black: $location`Vanya's Castle`,
+  blue: $location`Megalo-City`,
+  green: $location`Hero's Field`,
+  red: $location`The Fungus Plains`,
+};
+
+function value(color: BonusColor) {
+  const denominator = globalOptions.ascend
+    ? get("8BitScore") - FAT_LOOT_TOKEN_COST
+    : FAT_LOOT_TOKEN_COST;
+  return (garboValue($item`fat loot token`) * eightBitPoints(locationColor[color])) / denominator;
```

Comment:
pixel rock is 16k; is it worth accounting for that?
You can declare it out of scope if you want

Reply:
out of scope for this round of implementation